docs/site/index/process.md

```diff
@@ -24,7 +24,12 @@ The [UTS #35: Locale Data Markup Language (LDML)](https://www.unicode.org/report
 
 - Requests for changes are entered in the bug/feature request database ([file a ticket](/requesting_changes#how-to-file-a-ticket)).
 - Structural changes are always backwards-compatible. That is, previous files will continue to work. Deprecated elements remain, although their usage is strongly discouraged.
-- There is a standing policy for structural changes that require non-trivial code for proper implementation, such as time zone fallback or alias mechanisms. These require design discussions in the Technical Committee that demonstrates correct function according to the proposed specification.
+- There is a standing policy for structural changes that require non-trivial code for proper implementation, such as time zone fallback or alias mechanisms. These require design discussions in the [CLDR Design Working Group](https://cldr.unicode.org/cldr-tc/design-wg) and approval by the [CLDR Technical Committee](https://cldr.unicode.org/cldr-tc) that demonstrates correct function according to the proposed specification.
+- New sections may be added to the specification in Technical Preview, Final Candidate depending on how finalized the new section is, what type of feedback the Technical Committee requires, and whether the feedback period needs to extend across one or more releases.
+    - **Technical Preview** - Specification section have been approved by the CLDR Technical Committee and open for feedback. New features may be modified or removed similar to elements marked with [@TECHPREVIEW in the DTD as described in the LDML](https://www.unicode.org/reports/tr35/#dtd-annotations).
```

Comment:
Should be "Specification section **has** been approved" not "Specification section have been approved"? Sorry for missing this the first time around.

Reply:
Fixed. Please mark as reviewed if there aren't any further issues.